<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControllerWindow</class>
 <widget class="QMainWindow" name="ControllerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>389</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Egoboo Object Viewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="2">
     <widget class="QComboBox" name="comboBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>192</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QListWidget" name="listWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>192</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="3" rowspan="2">
     <widget class="ModelWidget" name="openGLWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>608</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpenRecent">
     <property name="title">
      <string>Open Recent</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actionClearRecent"/>
    </widget>
    <addaction name="actionOpen"/>
    <addaction name="menuOpenRecent"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionMinimize"/>
    <addaction name="actionZoom"/>
    <addaction name="separator"/>
    <addaction name="actionWindowSelect"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="toolTip">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionMinimize">
   <property name="text">
    <string>Minimize</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionZoom">
   <property name="text">
    <string>Zoom</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionWindowSelect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Egoboo Model Viewer</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt...</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
  <action name="actionClearRecent">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Clear Menu</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ModelWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>modelwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>listWidget</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>ControllerWindow</receiver>
   <slot>frameListChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>10</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ControllerWindow</receiver>
   <slot>skinChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>ControllerWindow</receiver>
   <slot>showAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>ControllerWindow</receiver>
   <slot>closeFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMinimize</sender>
   <signal>triggered()</signal>
   <receiver>ControllerWindow</receiver>
   <slot>doMinimize()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>ControllerWindow</receiver>
   <slot>openFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionWindowSelect</sender>
   <signal>triggered()</signal>
   <receiver>ControllerWindow</receiver>
   <slot>showWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoom</sender>
   <signal>toggled(bool)</signal>
   <receiver>ControllerWindow</receiver>
   <slot>doZoom()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClearRecent</sender>
   <signal>triggered()</signal>
   <receiver>ControllerWindow</receiver>
   <slot>clearRecentList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>frameListChanged()</slot>
  <slot>openFile()</slot>
  <slot>closeFile()</slot>
  <slot>doMinimize()</slot>
  <slot>doZoom()</slot>
  <slot>showAbout()</slot>
  <slot>showWindow()</slot>
  <slot>skinChanged(int)</slot>
  <slot>clearRecentList()</slot>
 </slots>
</ui>
